Integrations/integration-SymantecEndpointProtectionDeprecated_CHANGELOG_CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+## [Unreleased]
```

Comment:
`integration-SymantecEndpointProtectionDeprecated_CHANGELOG_CHANGELOG.md`? We doing changelogs for the changelogs now?

Reply:
oops...
I had a bug in my auto-RN script, thought I reverted them all